### PR TITLE
Update relative path that broke on cloud env

### PR DESCRIPTION
**Pull request by omar93**
# Description

The updated version of the app with relative path broke the ui and some cloud env, added an updated version of the absolute path with the windows check in place so it's platform agnostic

### Contribution Checklist:

- [x] **The pull request only addresses one issue or adds one feature.**
- [x] **The pull request does not introduce any breaking changes**
- [x] **I have added screenshots or gifs to help explain the change if applicable.**
- [x] **I have read the [contribution guidelines](https://github.com/usebruno/bruno/blob/main/contributing.md).**
- [x] **Create an issue and link to the pull request.**

Note: Keeping the PR small and focused helps make it easier to review and merge. If you have multiple changes you want to make, please consider submitting them as separate pull requests.

### Publishing to New Package Managers

Please see [here](../publishing.md) for more information.

image

### CODE REVIEW

**Comment by pietrygamat** [comment]
That's the exact opposite of #2420, which too has its merits. I suppose the only option to satisfy all parties is to add a toggle that allows saving relative or absolute paths depending on the user's decision. If tommorow I load your collection from a shared git repository and see path to your home directory, I am put back into the shoes you are in today.

Also, if the path to the certificate is variable between user PCs, and it is not stored in the repository together with the rest of the collection, I'd rather store it as a .env variable in the .bru file:
```
clientCertificates": {
    "enabled": true,
    "certs": [
      {
    "domain": "example.com",
        "certFilePath": "{{ certFilePath }}",
        "keyFilePath": "{{ keyFilePath }}",
        "passphrase": ""
      },
```

File: packages/bruno-app/src/components/CollectionSettings/ClientCertSettings/index.js [hunk 1]
```diff
@@ -128,6 +128,7 @@ const ClientCertSettings = ({ root, clientCertConfig, onUpdate, onRemove }) => {
       </ul>
 
       <h1 className="font-semibold mt-8 mb-2">Add Client Certificate</h1>
+      <h2>omar</h2>
```
Comment (JorgeTrovisco) [inline packages/bruno-app/src/components/CollectionSettings/ClientCertSettings/index.js:131]: Did you intend to leave this here?

**Comment by omar93** [comment]
that is true, i can remove the bug report and intead implement a feature that allows the user to pick relative / absolute path instead
________________________________
Från: Mateusz Pietryga ***@***.***>
Skickat: den 13 september 2024 16:31
Till: usebruno/bruno ***@***.***>
Kopia: Omar Alhamad ***@***.***>; Author ***@***.***>
Ämne: Re: [usebruno/bruno] Update relative path that broke on cloud env (PR #3092)


That's the exact opposite of #2420<https://github.com/usebruno/bruno/issues/2420>, which too has its merits. I suppose the only option to satisfy all parties is to add a toggle that allows saving relative or absolute paths depending on the user's decision. If tommorow I load your collection from a shared git repository and see path to your home directory, I am put back into the shoes you are in today.

Also, if the path to the certificate is variable between user PCs, and it is not stored in the repository together with the rest of the collection, I'd rather store it as a .env variable in the .bru file:

clientCertificates": {
    "enabled": true,
    "certs": [
      {
    "domain": "example.com",
        "certFilePath": "{{ certFilePath }}",
        "keyFilePath": "{{ keyFilePath }}",
        "passphrase": ""
      },

**Comment by omar93** [comment]
no, will remove it
________________________________
Från: JorgeTrovisco ***@***.***>
Skickat: den 14 september 2024 15:37
Till: usebruno/bruno ***@***.***>
Kopia: Omar Alhamad ***@***.***>; Author ***@***.***>
Ämne: Re: [usebruno/bruno] Update relative path that broke on cloud env (PR #3092)


@JorgeTrovisco commented on this pull request.

________________________________

In packages/bruno-app/src/components/CollectionSettings/ClientCertSettings/index.js<https://github.com/usebruno/bruno/pull/3092#discussion_r1759737595>:

> @@ -128,6 +128,7 @@ const ClientCertSettings = ({ root, clientCertConfig, onUpdate, onRemove }) => {
       </ul>

       <h1 className="font-semibold mt-8 mb-2">Add Client Certificate</h1>
+      <h2>omar</h2>


Did you intend to leave this here?

**Pull request closed without merging**